about.html

```diff
@@ -0,0 +1,136 @@
+          class="dropdown-menu display-non-mobile"
+        />
+        <div class="display-non-desktop disktop-nav">
+          <div class="close-btn">&#x2715;</div>
+          <a href="./index.html">
+            <img
+              src="./imges/icons/main-logo.svg"
+              alt="header-icon"
+              class="logo"
+            />
+          </a>
+          <ul>
+            <li>
+              <a href="about.html"> <span>About</span></a>
+            </li>
+            <li><a href="./index.html">program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">News</a></li>
+            <li><h4 class="box">camping</h4></li>
+          </ul>
+        </div>
+      </nav>
+      <div class="cover-about">
+        <h1>
+          CREATIVE COMMONS
+          <br />
+          GLOBAL SUMMIT 2021
+        </h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet
+          recusandae quia et? Ut cumque ullam magni Lorem ipsum dolor sit amet
+          consectetur adipisicing elit. Amet recusandae quia et? Ut cumque ullam
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet
+          recusandae quia et? Ut cumque ullam magni Lorem ipsum dolor sit amet
+          consectetur adipisicing elit. Amet recusandae quia et? Ut cumque ullam
+        </p>
+        <h3>2021.10.15</h3>
+        <h6>@ Lorem ipsum dolor sit amet consectetur</h6>
+      </div>
+    </section>
+    <section class="about">
+      <h2>this is all about us</h2>
+      <p>
+        all you need to know about us Lorem ipsum dolor sit amet consectetur
+        adipisicing elit. Amet recusandae quia et? Ut cumque ullam magni Lorem
+        ipsum dolor sit amet consectetur adipisicing elit. Amet recusandae quia
+        et? Ut cumque ullam magni
+      </p>
+      <img
+        src="./imges/icons/main-logo.svg"
+        alt=""
+        class="img-about-page-program"
+      />
+    </section>
+    <hr class="line1" />
+    <section class="see-past">
+      <h2>see the past</h2>
+      <hr class="main-program-line" />
+
+      <p>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Cupiditate
+        voluptatem temporibus ipsum natus esse suscipit tempora ab quasi
+        provident delectus. Perspiciatis voluptatem in accusamus alias ullam
+        provident sapiente, dicta nam!
+      </p>
+      <div class="imgs">
+        <div class="img-card">
+          <img src="./imges/about-img-1.jfif" alt="" />
+          <div class="about-year">
+            <h4>2020</h4>
+            <p>Lorem ipsum dolor sit</p>
+          </div>
+        </div>
+        <div class="img-card">
+          <img src="./imges/about-img-2.jfif" alt="" />
+          <div class="about-year">
+            <h4>2021</h4>
+            <p>Lorem ipsum dolor sit</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <hr class="line1" />
+
```

Comment:
[OPTIONAL]

 - I put this in optional because it was missed by the other code reviewers, but you need to follow the original layout. The below is the screenshot which you have missed in `about.html` mobile view. 
 

![Screenshot from 2021-07-03 23-19-33](https://user-images.githubusercontent.com/22434660/124362920-c1ff5880-dc55-11eb-8905-6f8600b35d29.png)